src/components/ListItem.jsx

```diff
@@ -1,9 +1,46 @@
 import './ListItem.css';
+import { useEffect } from 'react';
+import { updateItem } from '../api/firebase';
+
+export function ListItem({ listToken, item }) {
+	const millisecondsIn24hrs = 84600000;
```

Comment:
variables that remain unchanged like this should be defined at the top of the document otherwise they are are reassigned in a new memory bock everytime the component renders :)